python/ray/tune/suggest/bayesopt.py

```diff
@@ -121,35 +127,6 @@ def __init__(self,
                  analysis: Optional[ExperimentAnalysis] = None,
                  max_concurrent: Optional[int] = None,
                  use_early_stopped_trials: Optional[bool] = None):
-        """Instantiate new BayesOptSearch object.
-
-        Args:
-            space (dict): Continuous search space.
-                Parameters will be sampled from
-                this space which will be used to run trials.
-            metric (str): The training result objective value attribute.
-            mode (str): One of {min, max}. Determines whether objective is
-                minimizing or maximizing the metric attribute.
-            utility_kwargs (dict): Parameters to define the utility function.
-                Must provide values for the keys `kind`, `kappa`, and `xi`.
-            random_state (int): Used to initialize BayesOpt.
-            random_search_steps (int): Number of initial random searches.
-                This is necessary to avoid initial local overfitting
-                of the Bayesian process.
-            patience (int): Must be > 0. If the optimizer suggests a set of
-                hyperparameters more than 'patience' times,
-                then the whole experiment will stop.
-            skip_duplicate (bool): If true, BayesOptSearch will not create
-                a trial with a previously seen set of hyperparameters. By
-                default, floating values will be reduced to a digit precision
-                of 5. You can override this by setting
-                ``searcher.repeat_float_precision``.
-            analysis (ExperimentAnalysis): Optionally, the previous analysis
-                to integrate.
-            verbose (int): Sets verbosity level for BayesOpt packages.
-            max_concurrent: Deprecated.
-            use_early_stopped_trials: Deprecated.
-        """
```

Comment:
This is not needed (and out of date anyway) because we already have the class docstring

Reply:
great!